net/ddns-scripts/files/usr/lib/ddns/update_ns1_com.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+# Derived from update_gandi_net.sh
+
+. /usr/share/libubox/jshn.sh
+
+add_answer() { # {"answers":[{"answer":["1.1.1.1"]}]}
+	json_add_array answers
+		json_add_object
+			json_add_array answer
+				json_add_string "" "$1"
+			json_close_array
+		json_close_object
+	json_close_array
+}
```

Comment:
Please do not move in here, it will only confuse and if we need to change the object record every think gets changed.